tentacle/src/session.rs

```diff
@@ -580,7 +580,13 @@ impl Session {
                         } else {
                             buffer.push_normal(event)
                         }
-                        buffer.try_send(cx);
+                        if let SendResult::Pending = buffer.try_send(cx) {
+                            if self.context.pending_data_size() > self.config.send_buffer_size {
+                                self.state = SessionState::Abnormal;
+                                warn!("session {:?} unable to send message, and there are a lot of buffers on it, so kill it ", self.context);
```

Comment:
`
                                warn!("session {:?} unable to send message, and there are a lot of data in the buffer, so kill it ", self.context);
`